src/Uno.UI/UI/ColorHelper.cs

```diff
@@ -1,26 +1,102 @@
 ﻿using System;
 using System.ComponentModel;
-using Microsoft.UI;
+using System.Linq;
+using DirectUI;
+using WindowsColor = Windows/*Intentional space for WinUI upgrade tool*/.UI.Color;
 
-using Color = global::Windows/*Intentional space for WinUI upgrade tool*/.UI.Color;
+namespace Microsoft.UI;
 
-namespace Microsoft.UI
+public static partial class ColorHelper
 {
-	public static partial class ColorHelper
+	/// <summary>
+	/// Retrieves the display name of the specified color.
+	/// </summary>
+	/// <param name="color">The color to get the name for.</param>
+	/// <returns>The localized display name of the color.</returns>
+	public static string ToDisplayName(WindowsColor color)
 	{
-#if __ANDROID__ || __IOS__ || IS_UNIT_TESTS || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
-		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "IS_UNIT_TESTS", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
-		public static string ToDisplayName(Color color)
+		//uno specific
+
+		var closestKnownColor = Enum.GetValues<KnownColors>()
```

Comment:
Could you reference https://github.com/dotnet/wpf/pull/9661 and I think it be faster.

Reply:
Thanks @lindexi, we have actually reworked the implementation to be more inline with the WinUI side of things and to support localization. But thank you for your suggestion!